### PR TITLE
Feature/use spryker script for asset build

### PR DESCRIPTION
**Pull request by hgajjar-sessiondigital**

### CODE REVIEW

File: spryker/nginx/Dockerfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x > /tmp/install-node.sh \
     postgresql-client \
     redis-tools \
  \
+ && npm install -g antelope \
```
Comment (blongden) [inline spryker/nginx/Dockerfile]: Not needed again :)

File: spryker/nginx/usr/local/share/spryker/spryker_functions.sh [hunk 2]
```diff
@@ -24,12 +24,18 @@ do_spryker_config_create() {
...
   do_spryker_directory_create
   do_spryker_config_create
+  do_build_assets
 }
 
 do_build_assets() {
-  as_code_owner "npm install"
-  as_code_owner "npm run zed"
-  as_code_owner "npm run yves"
+  # use Spryker scripts to install static assets
```
Comment (blongden) [inline spryker/nginx/usr/local/share/spryker/spryker_functions.sh]: Has the regular npm install been run by this point?

File: spryker/nginx/Dockerfile (hunk 1 above)
Comment (hgajjar-sessiondigital) [inline spryker/nginx/Dockerfile]: we need this as the spryker script was complaining for this :)

Comment (blongden) [inline spryker/nginx/Dockerfile]: Ugh, really? 🤔

File: spryker/nginx/usr/local/share/spryker/spryker_functions.sh (hunk 2 above)
Comment (hgajjar-sessiondigital) [inline spryker/nginx/usr/local/share/spryker/spryker_functions.sh]: I think setupYvesFrontend does that

**Comment by blongden** [comment]
Looks good - @kierenevans could you merge please?

**Pull request merged**